theseus/src/config.rs

```diff
@@ -14,7 +14,7 @@ pub static REQWEST_CLIENT: Lazy<reqwest::Client> = Lazy::new(|| {
     reqwest::Client::builder()
         .tcp_keepalive(Some(time::Duration::from_secs(10)))
         .build()
-        .unwrap()
+        .expect("Reqwest Client Building Failed")
```

Comment:
We should add a UserAgent here (as per. labrinth specification), probably, if we plan to make fetches to the MODRINTH_API_URL. Don't want our own API to block us, lol